PatientWebApplication/HealthClinic.BL/Adapters/FeedbackAdapter.cs

```diff
@@ -1,8 +1,8 @@
-﻿using Class_diagram.Model.Patient;
-using PatientWebApplication.Dtos;
+﻿using HealthClinic.BL.Dtos;
+using HealthClinic.BL.Model.Patient;
 using System;
 
-namespace PatientWebApplication.Adapters
+namespace HealthClinic.BL.Adapters
 {
     public class FeedbackAdapter
     {
```

Comment:
Use constructor with parameters for new FeedbackDto in method FeedbackToFeedbackDto